src/screens/LocationDetailsScreen.tsx

```diff
@@ -1,19 +1,364 @@
-import { Text, View, StyleSheet } from "react-native";
+import { useCallback, useContext, useState } from "react";
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  RefreshControl,
+  Image,
+} from "react-native";
 
-export const LocationDetailsScreen = () => (
-  <View style={styles.container}>
-    <Text style={styles.huge}>Location Details Screen</Text>
-  </View>
-);
+import { VictoryBar, VictoryChart, VictoryAxis } from "victory-native";
+import { Button } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+import { ThemeContext } from "@contexts/theme";
+import { StyledText } from "@components/StyledText";
+import { SuccessModal } from "@components/SuccessModal";
+import { renderLastUpdated } from "@utils/waitTime";
+
+export const LocationDetailsScreen = () => {
+  const { theme } = useContext(ThemeContext);
+  const [refreshing, setRefreshing] = useState(false);
+  const [showSubmittedSuccessModal, setSubmittedSuccessModal] = useState(false);
+  const [showConfirmSuccessModal, setConfirmSuccessModal] = useState(false);
+  const [newWaitTime, setNewWaitTime] = useState(0);
+
+  const chartData = [
+    { quarter: 7, earnings: 1 },
+    { quarter: 8, earnings: 2 },
+    { quarter: 9, earnings: 1 },
+    { quarter: 10, earnings: 4 },
+    { quarter: 11, earnings: 6 },
+    { quarter: 12, earnings: 15 },
+    { quarter: 13, earnings: 6 },
+    { quarter: 14, earnings: 4 },
+    { quarter: 15, earnings: 5 },
+    { quarter: 16, earnings: 5 },
+    { quarter: 17, earnings: 7 },
+    { quarter: 18, earnings: 3 },
+    { quarter: 19, earnings: 15 },
+    { quarter: 20, earnings: 3 },
+    { quarter: 21, earnings: 2 },
+    { quarter: 22, earnings: 5 },
+    { quarter: 23, earnings: 1 },
+  ];
+
+  const poiData = {
+    name: "Centro",
+    address: "Commons Building",
+    distance: 2.1,
+    hours: [
+      { "Monday - Friday": "7:00 AM - 11:00 PM" },
+      { "Saturday & Sunday": "10:00 AM - 11:00 PM" },
+    ],
+    lastUpdated: 4,
+    waitTime: 10,
+    currentHour: 14,
+  };
+
+  const submitNewWaitTime = () => {
+    // send new wait time to backend
+    // api call with newWaitTime
+    setSubmittedSuccessModal(true);
+    setTimeout(() => {
+      setSubmittedSuccessModal(false);
+    }, 2000);
+  };
+
+  const confirmNewWaitTime = () => {
+    // send wait time to backend
+    setConfirmSuccessModal(true);
+    setTimeout(() => {
+      setConfirmSuccessModal(false);
+    }, 2000);
+  };
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 1000);
+  }, []);
+
+  return (
+    <View style={theme.styles.screenContainer}>
+      <ScrollView
+        style={styles.scrollContainer}
```

Comment:
To fix the padding issue you'll need to set the topPadding to 0 to override the screenContainer 
```
<View style={[theme.styles.screenContainer, styles.noTopPadding]}>
```
Then add to scrollContainer a topPadding value of like 20.